ding/utils/log_helper.py

```diff
@@ -32,10 +32,23 @@ def build_logger(
         name = 'default'
     logger = LoggerFactory.create_logger(path, name=name) if need_text else None
     tb_name = name + '_tb_logger'
-    tb_logger = DistributedWriter(os.path.join(path, tb_name)) if need_tb else None
+    tb_logger = TBLoggerFactory.get_logger(os.path.join(path, tb_name)) if need_tb else None
     return logger, tb_logger
 
 
+class TBLoggerFactory(object):
+
+    tb_loggers = {}
+
+    @staticmethod
+    def get_logger(logdir: str) -> DistributedWriter:
+        if logdir in TBLoggerFactory.tb_loggers:
```

Comment:
Since you have already use `TBLoggerFactory.tb_loggers` class variable here, I think this method it's better to set this method as `classmethod`

Reply:
done